app/src/main/java/com/naxtlevelofandroiddevelopment/marvelgallery/network/MarvelApi.kt

```diff
@@ -0,0 +1,17 @@
+package com.naxtlevelofandroiddevelopment.marvelgallery.network
```

Comment:
When there is single class for each layer you do not need packages at all. Just put classes in domain package like `movie`, `comic`, `film`, `order`